caasp-istio-proxyv2-image/_service

```diff
@@ -0,0 +1,10 @@
+<services>
+    <service name="replace_using_package_version" mode="buildtime">
```

Comment:
The same as above, this service makes no sense if you hardcode the tags. Note the this service can query the version of any of the packages installed within the image, even if they are not listed in the description file (not sure if this is the case here).

Reply:
the image will now be reverted to use %%PKG_VERSION%% and the cilium-istio-proxy package is now being modified to build as version 1.5.4. So this hard-coded tagging can be removed and _service file can be retained.